vita3k/renderer/src/gl/renderer.cpp

```diff
@@ -486,9 +488,15 @@ void get_surface_data(GLContext &context, size_t width, size_t height, size_t st
         glReadPixels(0, 0, static_cast<GLsizei>(width), static_cast<GLsizei>(height), GL_RED, GL_UNSIGNED_BYTE, pixels);
         break;
     case SCE_GXM_COLOR_FORMAT_U2F10F10F10_ABGR:
+    case SCE_GXM_COLOR_FORMAT_F10F10F10U2_RGBA:
     case SCE_GXM_COLOR_FORMAT_U2U10U10U10_ABGR:
+    case SCE_GXM_COLOR_FORMAT_U10U10U10U2_RGBA:
```

Comment:
Same problem here, you have different order all using the same readback settings making it impossible for all of them to be right.

Reply:
I have explain all in dm obligatory need use same value for works

Reply:
It is impossible for both to be right, so if adding this fixes something there are 2 possibilities:
1. The existing format _ABGR is wrong.
2. There is a bug somewhere else that is making this work.
I get the need for some results, but this doesn't fix the real problem. Someone will have to redo all of these properly later.